src/components/SortBy/SortBy.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styles from './sortby.scss';
+
+const SortBy = () => (
```

Comment:
Could you please make SortBy component to accept callback `onHandleSort` and call it when changing sorting filter